### DISCUSSION TITLE
Configuring --dashboard --html=report.html in addopts of pytest.ini cannot produce advanced html reports

### DISCUSSION THREAD

**Discussion by HorsonA** [seleniumbase/SeleniumBase#1378] [Q&A] [answered]
Configuring --dashboard --html=report.html in addopts of pytest.ini cannot produce advanced html reports.
I must enter pytest --dashboard --html=report.html on the command line to generate advanced html reports.
My seleniumbase version is 3.3.0.

**Comment by mdmintz** [comment]
I would check to see that you aren't using multiple ``pytest.ini`` files. I used one, and it worked for me.

```ini
[pytest]

# Display console output, disable cacheprovider, and have the ipdb debugger replace pdb:
addopts = --capture=no -p no:cacheprovider --pdbcls=IPython.terminal.debugger:TerminalPdb --dashboard --html=report.html

# Ignore warnings such as DeprecationWarning and PytestUnknownMarkWarning
filterwarnings =
    ignore::pytest.PytestWarning
    ignore:.*U.*mode is deprecated:DeprecationWarning
```

Here's the output from running a test:

```bash
pytest my_first_test.py
========================================== test session starts ===========================================
platform darwin -- Python 3.10.1, pytest-7.1.2, pluggy-1.0.0
rootdir: /Users/michael/github/SeleniumBase/examples, configfile: pytest.ini
plugins: html-2.0.1, xdist-2.5.0, forked-1.4.0, metadata-2.0.1, rerunfailures-10.2, ordering-0.6, cov-3.0.0, seleniumbase-3.3.2
collected 1 item                                                                                         
Dashboard: /Users/michael/github/SeleniumBase/examples/dashboard.html
*********************************************************************

my_first_test.py .

---------- generated html file: file:///Users/michael/github/SeleniumBase/examples/report.html -----------
=========================================== 1 passed in 8.50s ============================================
```

**Reply by HorsonA** [reply]
This is my project structure:
![2566ea2b9e5482d75eb4437634beae9](https://user-images.githubusercontent.com/108264903/176136087-360eafcf-aa58-4cee-8458-6009befd24e8.png)

Using pytest from the command line:
![420b2df06f3d69371f0893af9f5371c](https://user-images.githubusercontent.com/108264903/176136152-e5e8ded1-5cf2-4400-823c-6a9518872048.png)
reprot.html shows:
![6b50d870fdbd2e6694675a1c651828a](https://user-images.githubusercontent.com/108264903/176136181-c26830b7-8bab-434d-b991-d33056191785.png)

However when I use pytest --dashboard --html=report.html on the command line:
![5e642ac7ff42ee216959c8289a6b338](https://user-images.githubusercontent.com/108264903/176136231-1723a93b-32ec-47f6-92af-e6fccf64f454.png)
reprot.html shows:
![e50a1b03f6db65cbdd38e76abe97abf](https://user-images.githubusercontent.com/108264903/176136260-190bc3e7-5136-432f-a6f9-fe6aa6ae89a5.png)

**Reply by mdmintz** [reply]
I created https://github.com/seleniumbase/SeleniumBase/issues/1379 for the issue. A fix will be ready shortly.

**Comment by mdmintz** [comment, marked as answer]
This has been fixed in https://github.com/seleniumbase/SeleniumBase/releases/tag/v3.3.3

**Reply by HorsonA** [reply]
thanks!
